cross-modal-search/README.md

```diff
@@ -46,11 +34,11 @@ for them to have different modalities.
 the body text and the title are from the same mime type (text),
 but can be considered of different modalities (distributions).
 
-Different encoders map different modalities to a common embedding space.
+Different Encoders map different modalities to a common embedding space.
 They need to extract semantic information from the documents. 
 
 In this embedding space,
-documents that are semantically relevant to each other from different modalities are expected to be close to another -  Metric Learning
+documents that are semantically relevant to each other from different modalities are expected to be close to another - Metric Learning
```

Comment:
Can you define "Metric Learning" or set a link for description?

Reply:
5f33ee0ac0a037bf57c8c44113a7548b79a90bf5